Angular/Server-side rendering (SSR)/Environment variables.md

```diff
@@ -1,5 +1,5 @@
 When using SSR, the node server can read system environment variables at runtime and transfer them to the client using
-the [`TransferState` API](https://angular.io/api/platform-browser/TransferState). This is just one of the use cases for
+the [`TransferState` API](https://angular.dev/api/core/TransferState). This is just one of the use cases for
 the `TransferState` API.
 
 To make working with environment variables in an SSR application easier, we have created `EnvironmentVariablesSSRLoader`
```

Comment:
EnvironmentVariablesSSRLoader no longer exists and the code configuration below is outdated. Most of this file can be deleted and just link to [Env vars SSR chapter from ngx-nuts-and-bolts](https://infinum.github.io/ngx-nuts-and-bolts/docs/environment-variables#232-for-ssr--angular-universal-apps)